layouts/partials/misc/contacts.html

```diff
@@ -2,15 +2,15 @@
     <div class="d-flex flex-column flex-md-row gap-4">
         <div class="col col-md-5 d-flex flex-column gap-4 flex-grow-1 jusitfy-space-between">
```

Comment:
can you please change `jusitfy-space-between` to `justify-content-between`? It will properly align the three cards with the contact form on the right